examples/kitchensink/server.go

```diff
@@ -26,6 +26,7 @@ import (
 	"path/filepath"
 	"runtime"
 
+	"github.com/line/line-bot-sdk-go/v8/linebot"
```

Comment:
```suggestion
	"github.com/line/line-bot-sdk-go/v8/linebot/util"
```
Sorry, we'd like to delete `github.com/line/line-bot-sdk-go/v8/linebot` from v9. Could you rename this as `until` (or something like that) and move `find_dollar_sign.go` to the directory?

(`find_doller.sign.go`)
```diff
-package linebot
+package util
```

Reply:
Updated